sources/main.cpp

```diff
@@ -76,17 +79,21 @@ main(int argc, char* argv[]) {
     if (!boost::filesystem::is_regular_file(dictPath)) {
       // we cannot use dictionnary errors since they are not be initialized yet
       LOG(error) << "Dictionary " << dictPath << " not found: check runtime environment" << LOG_ENDL;
+      std::cerr << "Dictionary " << dictPath << " not found: check runtime environment" << std::endl;
```

Comment:
why use cerr ? Don't we want to use the log instead ? cerr should be used only when it is unsure that log file is initialized

Reply:
It find it nicer to have a feedback in the terminal when an error happened, but that's only personal
@marcochiaramello an opinion on this?

Reply:
I do agree, maybe in debug mode if possible..otherwise I'll keep it

Reply:
In that case, it should be done in dynawo, in function error from Trace

Reply:
ok I removed them